### DISCUSSION TITLE
Use Android PWA Without Chrome

### DISCUSSION THREAD

**Discussion by JPDVM2014** [schlagmichdoch/PairDrop#119] [Q&A]
Is it possible to install the PWA, with share menu functionality, without using Chrome?

I can install using Firefox and Vivaldi, but it doesn't show in the share menu. I prefer to keep Chrome disabled if possible. Thanks!

**Comment by schlagmichdoch** [comment]
Good question!
While the [official article](https://developer.chrome.com/articles/web-share-target/) (obviously) only mentions Chrome I believe you can also use Samsung Internet or Opera Mobile. At least that is what caniuse.com states: https://caniuse.com/?search=manifest%20share_target

I'll test this. Feel free to add your findings here. I guess I will add this question to the FAQ afterwards.

**Reply by JPDVM2014** [reply]
I tried with Samsung browser, but no luck.

So far, the best option besides Chrome seems to be this Snapdrop app - https://github.com/fm-sys/snapdrop-android.

It seems to work fine from the share menu and even when not on the same network.

**Reply by schlagmichdoch** [reply]
So I did some tests too. Opera did not work.
Samsung Internet did indeed work so this would be the answer to your question. To install PairDrop as a webapp, avoid "Add page to homescreen" and use one of these two buttons instead:
<image src="https://github.com/schlagmichdoch/PairDrop/assets/52242352/29f72df9-8aa9-4662-9f50-8a0fc71ba3f9" width=250/>

The usage of the Android app by fm-sys is a good option too though. Just use what's best for you.

**Reply by JPDVM2014** [reply]
Well, that explains why Samsung browser didn't work. Thanks for getting back to me, and for Pairdrop!
